src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -262,6 +262,9 @@ if [[ -n $PAI_CODE_DIR ]]; then
   if [ $code_dir_size -gt 0 ] && [ $code_dir_size -le 200000000 ]; then
```

Comment:
Since the code_dir will be removed in the future. Should we pass the size==0 condition instead of report error?

Reply:
yes size==0 should be allowed.

Reply:
ok, I will make some changes corresspondingly.